src/documentation/markdown.rs

```diff
@@ -71,15 +71,46 @@ impl<'a> PlainOverlay<'a> {
                                 // @todo validate as if it was another document entity
                             }
                         }
+                        Tag::Link(link_type, _url, title) => {
+                            // @todo check links
+                            // for now, only dealing with some links types
+                            match link_type {
+                                LinkType::Inline => {}
+                                LinkType::Autolink | LinkType::Email => skip_text = true,
+                                //Reference,
+                                //ReferenceUnknown,
+                                //Collapsed,
+                                //CollapsedUnknown,
+                                //Shortcut,
+                                //ShortcutUnknown,
+                                _ => {}
+                            }
+                        }
 
                         _ => {}
                     }
                 }
                 Event::End(tag) => {
                     match tag {
-                        Tag::Link(_link_type, _url, title) => {
-                            // @todo check links
-                            Self::track(&title, offset, &mut plain, &mut mapping);
+                        Tag::Link(link_type, _url, title) => {
+                            // for now, only dealing with some links types
+                            match link_type {
+                                // todo:
+                                LinkType::Inline => {
+                                    if !title.borrow().is_empty() {
```

Comment:
oki, seems I have to revert this:


`method not found in `pulldown_cmark::strings::CowStr<'_>``